snewpdag/plugins/gen/GenTimeDist.py

```diff
@@ -106,9 +120,9 @@ def alert(self, data):
         dt = self.tedges[j+1] - ta
         a = ta + Node.rng.random(nev) * dt
 
-      # add offsets in seconds - works for TimeHist or TimeSeries
+      # add offsets in seconds - works for Hist1D or TimeSeries
       a += offset
-      v.add_offsets_s(a)
+      v.add(a)
```

Comment:
Since going to float timestamps and removing the reference time stuff from the time histogram and series objects (which meant that TimeHist didn't really add anything to Hist1D, which is why it disappeared), we only add timestamps.  Would it help to change to add_times() I suppose, just to make it explicit even though there's one type of input?